index.html

```diff
@@ -10439,23 +10491,34 @@ <h2 id="WaveShaperNode">
               sequence&lt;float&gt; real
             </dt>
             <dd>
-              The array of <code>cosine</code> terms equivalent to the
-              <a><code>real</code></a> parameter to <a href=
-              "#widl-BaseAudioContext-createPeriodicWave-PeriodicWave-Float32Array-real-Float32Array-imag-PeriodicWaveConstraints-constraints">
-              <code>createPeriodicWave</code></a>. This defaults to a sequence
-              of all zeroes of the same length as <a><code>imag</code></a> if
-              <a><code>imag</code></a> is given.
+              <p>
+                The <dfn id="dfn-real">real</dfn> parameter represents an array
+                of <code>cosine</code> terms (traditionally the A terms). In
```

Comment:
I think it's best to leave off the "(traditionally the A terms)".  Or perhaps better just say this represents the a[k] terms in the waveform generation section.

Likewise for the B terms below when discussing the imag element.